src/tag.c

```diff
@@ -32,6 +32,8 @@ typedef struct tag_pointers
 #endif
     char_u	*tagkind;	/* "kind:" value */
     char_u	*tagkind_end;	/* end of tagkind */
+    char_u	*tagline;	/* "line:" value */
+    char_u	*tagline_end;	/* end of tagline */
```

Comment:
`tagline_end` is not really used, but just `atoi`.
Maybe `tagline_end` could be used.